tutorials/2d/2d_sprite_animation.rst

```diff
@@ -240,7 +240,7 @@ released.
             _animationPlayer = GetNode<AnimationPlayer>("AnimationPlayer");
         }
 
-        public override void _Process(float _delta)
+        public override void _Process(double _delta)
```

Comment:
The `delta` parameter name should match the base declaration, even if unused. Sorry that I didn't mention this before, I missed it.

```suggestion
        public override void _Process(double delta)
```